logging/bin/deploy_logging_open_openshift.sh

```diff
@@ -127,7 +127,14 @@ logging/bin/deploy_servicemonitors_open_openshift.sh
 # Version Info                   #
 ##################################
 log_info "STEP 7: Updating version info"
-if ! deployV4MInfo "$LOG_NS" "v4m"; then
+
+# If a deployment with the old name exists, remove it first
+if helm3ReleaseExists v4m $LOG_NS; then
+  log_verbose "Removing outdated instance of SAS Viya Monitoring"
```

Comment:
How about changing the message to "Removing outdated SAS Viya Monitoring Helm chart release from [$LOG_NS] namespace"?

Reply:
Updated the message per your recommendation for all deployment scripts.  It should show up in the next push.